packages/plugins/rows/src/editor/separator.tsx

```diff
@@ -79,16 +90,23 @@ export function Separator({
   isFirst,
   onClick,
   focused,
+  visuallyEmphasizedAddButton = false,
 }: {
   config: RowsPluginConfig
   isFirst?: boolean
   onClick: () => void
   focused?: boolean
+  visuallyEmphasizedAddButton?: boolean
 }) {
   return (
     <StyledSeparator isFirst={isFirst}>
```

Comment:
Even though the styling is all right for now, I'd love some top-margin.
Cleanest way I see is to give the last `StyledSeparator` `positition: static` (or not set `absolute`) with something like:

``` jsx
 <StyledSeparator isFirst={isFirst} isLast={isLast} >
 ```

Reply:
Something like that would work nicely

<img width="603" alt="image" src="https://user-images.githubusercontent.com/1258870/225904483-8d4e8a4c-27cf-4a4b-8a0f-9de2a2192686.png">

``` css
{
    position: static;
    margin-bottom: 5rem;
    margin-top: -3rem;
}
```

Reply:
Thanks for looking into it. I already implemented it in a different way however.